lib/opentok.js

```diff
@@ -199,6 +199,11 @@ OpenTok = function (apiKey, apiSecret, env) {
   *          {@link OpenTok#removeArchiveStream OpenTok.removeArchiveStream()} methods.</li>
   *       </ul>
   *   </li>
+  *   <li>
+  *     <code>multiArchiveTag</code> (String) &mdash; The tag of the archive, with which to start, 
+  *     this is unique for each archive within session. <code>multiArchiveTag</code> is not 
+  *     case-sensitive (e.g. 'MyArchive' and 'myarchive' are treated as same)
+  *   </li>
```

Comment:
This is not quite accurate. See the docs for `multiArchiveTag` in the REST API docs: https://tokbox.com/developer/rest/#start_archive.